rules/windows/process_creation/proc_creation_win_lolbas_not_from_c_drive.yml

```
@@ -7,14 +7,18 @@ date: 2022/01/25
 modified: 2022/02/14
 references: 
     - https://thedfirreport.com/2021/12/13/diavol-ransomware/
+    - https://www.scythe.io/library/threat-emulation-qakbot
 tags:
     - attack.t1218.001
 logsource:
     category: process_creation
     product: windows
 detection:
     selection:
-        Image|endswith: '\rundll32.exe'
+        Image|endswith: 
+        - '\rundll32.exe'
+        - '\calc.exe'
+        - '\regsvr32.exe'
```

```suggestion
            - '\rundll32.exe'
            - '\mshta.exe'
            - '\cscript.exe'
            - '\wscript.exe'
            - '\regsvr32.exe'
            - '\installutil.exe'
            - '\cmstp.exe'
```
Not sure if calc would be abused outside PoCs, but maybe you can include a few more. Not sure if simple to define in the condition, but looking for those with 2>= args would reduce the FPs a lot

---

As a side note, it's good practice to add  `calc` in my opinion. Since it's often ignored which leads to unwanted bypasses. Also, there are a couple of real-world examples where threat actors used calc as a way to bypass detections. So I think it's worth keeping.

Unfortunately currently, there is no way to express the number of arguments present in the command line. Except with a regex (which is discouraged).

---

Sure, just not really seems to fit in the logic, not a lolbas (AFAIK)... And it is a attacker-controlled variable, so it can be anything, like `w0rk3r.exe`. Maybe a dedicated rule for unsigned calc would do a better job

---

Calc has been observed being abused by Qakbot. Thanks for adding more, I'm busy but at least wanted to get those I've seen recently in CTI.
https://www.bleepingcomputer.com/news/security/qbot-phishing-uses-windows-calculator-dll-hijacking-to-infect-devices/

---

If they rename an application the rule below should trip. I see two ways. Either add calc to the rename rule below, or add original file name section to this rule. 

https://github.com/SigmaHQ/sigma/blob/8e12115516ea88c2ea2c72e33e5af0cc5d8f1f93/rules/windows/process_creation/proc_creation_win_renamed_binary.yml

---

It's not exactly a LOLBAS, as it's just DLL Side-loading w/o extra features or functionality, but seemed to make sense instead of writing another rule 🤷 Let me know how you all want to address it.